.github/dependabot.yml

```diff
@@ -14,7 +14,15 @@ updates:
     - "type/dependencies"
     - "scope/backend"
   groups:
-    gradle-dependencies:
+    spring-boot-dependencies:
+      patterns:
+        - "org.springframework.boot:*"
+        - "io.spring.dependency-management"
+      # We will handle major upgrades manually
+      update-types:
+        - "patch"
+        - "minor"
```

Comment:
I copied this from the other entries but I don't agree necessarily

Reply:
Maybe it makes sense to indeed have that group (via separate PRs) so we'll at least know there's a new version out quicker